<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="46086" started-at="2019-02-05T12:25:51Z" finished-at="2019-02-05T12:26:37Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="46086" started-at="2019-02-05T12:25:51Z" finished-at="2019-02-05T12:26:37Z">
      <class name="com.towerhobbies.HomePage.HomePage">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:com.towerhobbies.HomePage.HomePage@553a3d88]" name="beforeTest" is-config="true" duration-ms="21072" started-at="2019-02-05T12:25:51Z" finished-at="2019-02-05T12:26:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="QuickInfo()[pri:0, instance:com.towerhobbies.HomePage.HomePage@553a3d88]" name="QuickInfo" duration-ms="24991" started-at="2019-02-05T12:26:12Z" finished-at="2019-02-05T12:26:37Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Title is correct expected [Einkaufskrb] but found [Einkaufskorb]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Title is correct expected [Einkaufskrb] but found [Einkaufskorb]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.towerhobbies.HomePage.RegistrationPage._quickInfo(RegistrationPage.java:226)
at com.towerhobbies.HomePage.HomePage.QuickInfo(HomePage.java:53)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- QuickInfo -->
      </class> <!-- com.towerhobbies.HomePage.HomePage -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
